include/ck/tensor_operation/gpu/device/device_binary_elementwise_2d.hpp

```diff
@@ -0,0 +1,202 @@
+#pragma once
+#include <iostream>
+#include <vector>
+
+#include "device.hpp"
+#include "device_binary_elementwise.hpp"
+#include "gridwise_binary_elementwise_1d.hpp"
+
+namespace ck {
+namespace tensor_operation {
+namespace device {
+
+template <typename ADataType,
+          typename BDataType,
+          typename CDataType,
+          typename ComputeDataType,
+          typename ElementwiseFunctor,
+          index_t ThreadPerBlock,
+          index_t ScalarPerVector>
+struct DeviceBinaryElementwise_2D : public DeviceBinaryElementwise<ElementwiseFunctor>
```

Comment:
DeviceBinaryElementwise_ND

You could make this Device Operation supporting N-D tensor (N=1~5).

Reply:
Please open a JIRA task ticket and a github issue, and refer to this comment in both tickets.

Reply:
Please open a JIRA task ticket and a github issue, and refer to this comment in both tickets.

Reply:
https://ontrack-internal.amd.com/browse/LWPCK-190

Reply:
https://github.com/ROCmSoftwarePlatform/composable_kernel/issues/203